Source/SwiftLintFramework/Rules/StatementPositionRule.swift

```diff
@@ -41,19 +42,85 @@ public struct StatementPositionRule: CorrectableRule, ConfigurationProviderRule
         ]
     )
 
+    public static let uncuddledDescription = RuleDescription(
+        identifier: "statement_position",
+        name: "Statment Position",
+        description: "Else and catch should on the next line, with equal indentation to the " +
+        "previous declaration.",
```

Comment:
could you please indent this string continuation? Just like the previous `description` above.